template-tags.php

```diff
@@ -417,22 +417,61 @@ function coauthors_ids( $between = null, $betweenLast = null, $before = null, $a
 	), null, $echo );
 }
 
-function get_the_coauthor_meta( $field ) {
-	global $wp_query, $post;
+/**
+ * Outputs the co-authors Meta Data
+ *
+ * @param string $field Required The user field to retrieve.[login, email, nicename, display_name, url, type]
+ * @param string $user_id Optional The user ID for meta
+ *
+ * @return array $meta Value of the user field
+ */
+function get_the_coauthor_meta( $field, $user_id = false ) {
+	global $coauthors_plus;
+
+	if ( ! $user_id ) {
+		$coauthors = get_coauthors();
+    } else {
+        $coauthor_data = $coauthors_plus->get_coauthor_by( 'id', $user_id );
+        $coauthors = array();
+        if ( ! empty( $coauthor_data ) ) {
+            $coauthors[] = $coauthor_data;
+        }
+    }
 
-	$coauthors = get_coauthors();
 	$meta = array();
 
+    if ( in_array( $field, array( 'login', 'pass', 'nicename', 'email', 'url', 'registered', 'activation_key', 'status' ) ) ) {
+        $field = 'user_' . $field;
+    }
+
 	foreach ( $coauthors as $coauthor ) {
 		$user_id = $coauthor->ID;
-		$meta[ $user_id ] = get_the_author_meta( $field, $user_id );
+
+        if ( isset( $coauthor->type )  && 'user_url' === $field ) {
```

Comment:
Spacing looks a little off from here to line 462 as well. Perhaps spaces vs tabs?

Reply:
@philipjohn : I have made space alignment changes here. Could you please check and let me know if everything is ok?